scss/components/_cookies-banner.scss

```diff
@@ -68,4 +68,17 @@
         padding: 0;
         margin: 8px 0;
     }
+
+    &__preferences-success {
+        display: block;
+        border-left: 7px solid $salem;
```

Comment:
As discussed 7px doesn't really fit in with the grid and should be using the `$col` value multiplied by how many columns there are which this space occupies (so anything with a factor of 4)